src/components/Modal/ModalShare/ModalShare.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styled from "styled-components";
+import imageData from "../../../assets/imageData";
+import {
+  ModalFolderTitle,
+  ModalTitleContainer,
+  ModalTitle,
+  ModalWrapper,
+} from "../Modal.styled";
+import ModalShareLinkIcon from "./ModalShareLinkIcon";
+import { handleCopyClipBoard } from "../../../utils/util";
+import { shareKakao } from "../../../utils/kakaoShare";
+
+export default function ModalShare({ folderName = "", itemList, userId }) {
+  const [curFolder] = itemList.filter((item) => item.name === folderName);
```

Comment:
`const [curFolder] = itemList.filter((item) => item.name === folderName);` 는 itemList.filter((item) => item.name === folderName);를 해서 반환된 배열의 "첫번째 요소"를 curFolder라는 변수명으로 할당하는 것입니다.
이 코드를 읽는 개발자에게는 아마도 filter로 반환된 배열의 "첫번째 요소" 외에 "나머지 다른 요소"들의 존재가 신경쓰일 수 있어요. "만약 나머지 다른 요소들이 있다면 왜 다른 요소들은 사용하지 않고 첫번째 요소만 curFolder라는 이름을 붙여서 사용하는 것일까?" 같이요.

그래서 아래와 같이 배열의 find 메서드를 사용해서 리팩터링하면 어떨까요? 둘 다 결과적으로 같은 효과의 코드이지만, 의도가 더 명확하게 드러나는 코드는 어떤 것일지 고민하면 좋습니다~! 
```suggestion
  const curFolder = itemList.find((item) => item.name === folderName);
```

Reply:
그리고 curFolder라는 이름처럼 축약된 이름보다는 되도록 currentFolder 처럼 맞춤법에 맞는 네이밍을 하면 좋겠습니다. 

코드에서 축약어는 일반적으로 두 가지 위험을 초래합니다.
- 코드를 읽는 사람이 축약어를 이해하지 못할 수 있습니다.
- 다른 개발자가 같은 단어에 대해 여러 가지 다른 축약어를 사용해 쓸데없는 혼란을 초래할 수 있습니다.

Reply:
[refactor : filter를 find로 변경 및 변수명 변경](https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/pull/227/commits/92bfa62f9ad6ad4f768a6cd8a585dbdd10d79b6c)
여기서 수정했습니다!